.github/workflows/build_images.yml

```diff
@@ -50,13 +49,15 @@ jobs:
 
       - name: Build and test front docker image
         run: |
-          pip install wait-for-it
+          pip install wait-for-it pytest pytest-cov
```

Comment:
Does this repeat what `pip install -e .` does later (line 59)?

Reply:
I wasn't previously aware that dev dependencies got pulled in by default. Makes sense now so have removed this duplicate install, and at the same time done the same for the feedbackapi. (added dev deps in setup.py and removed duplicate pytest install)